blocks/inspector-controls/base-control/index.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+function BaseControl( { id, label, classes, children, ...props } ) {
```

Comment:
Should we rename `classes` as `className` for consistency with React Props and our other components?